src/osa/configs/sequencer.cfg

```diff
@@ -15,9 +15,10 @@ RUN_SUMMARY_DIR: %(MONITORING)s/RunSummary
 RUN_CATALOG: %(MONITORING)s/RunCatalog
 PEDESTAL_FINDER_DIR: %(BASE)s/auxiliary/PedestalFinder
 ANALYSIS_DIR: %(BASE)s/running_analysis
-CALIB_BASE_DIR: %(MONITORING)s/PixelCalibration/Cat-A
-CALIB_DIR: %(CALIB_BASE_DIR)s/calibration
-PEDESTAL_DIR: %(CALIB_BASE_DIR)s/drs4_baseline
+CALIB_BASE_DIR: %(MONITORING)s/PixelCalibration
+CALIB_DIR: %(CALIB_BASE_DIR)s/Cat-A/calibration
+PEDESTAL_DIR: %(CALIB_BASE_DIR)s/Cat-A/drs4_baseline
+CATB_CALIB_DIR: %(CALIB_BASE_DIR)s/Cat-B
```

Comment:
```suggestion
CALIB_BASE_DIR: %(MONITORING)s/PixelCalibration
CAT_A_CALIB_BASE: %(CALIB_BASE_DIR)s/Cat-A
CAT_A_CALIB_DIR: %(CAT_A_CALIB_BASE)s/calibration
CAT_A_PEDESTAL_DIR: %(CAT_A_CALIB_BASE)s/drs4_baseline
CAT_B_CALIB_BASE: %(CALIB_BASE_DIR)s/Cat-B
```

Reply:
I would specify each Cat A and Cat B name in the directories. It has to be change in the code accordingly